webapps-schema/src/main/java/io/camunda/webapps/schema/entities/operate/ProcessEntity.java

```diff
@@ -71,7 +75,9 @@ public boolean equals(final Object o) {
         && Objects.equals(bpmnXml, that.bpmnXml)
         && Objects.equals(resourceName, that.resourceName)
         && Objects.equals(flowNodes, that.flowNodes)
-        && Objects.equals(tenantId, that.tenantId);
+        && Objects.equals(tenantId, that.tenantId)
+        && Objects.equals(formId, that.formId)
+        && Objects.equals(isPublic, that.isPublic);
```

Comment:
Are we ready to come with the public information over here? Or will be implicit for the future? Once it comes related to authentication (not previewed from 8.7 for this topic)

Reply:
This is just information extracted from the BPMN file, as it was before over on the `tasklist-process` index. I believe accessing public processes will be bound to the authentication changes so as to actually enable this field.